### ISSUE TITLE
Problem with StreamedResponse

### ISSUE THREAD

**Issue by vsychov** [Baldinof/roadrunner-bundle#9]
Hi,

I found that [StreamedResponse](https://symfony.com/doc/master/components/http_foundation.html#streaming-a-response) not work now, example code for reproduce problem:

```
<?php

namespace App\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\StreamedResponse;
use Symfony\Component\Routing\Annotation\Route;

class TestController extends AbstractController
{
    /**
     * @Route("/test")
     */
    public function test()
    {
        $response = new StreamedResponse(null, 200, ['Content-Type' => 'text/plain']);
        $response->setCallback(function () {
            var_dump('Hello World');
            flush();

            var_dump('Hello World');
            flush();
        });

        return $response;
    }
}

```

**Comment by vsychov** [comment]
This issue may be fixed by #10, but this fix looks not well.

**Comment by vsychov** [comment]
Another way - disallow usage of StreamedResponse, and create some alternative in this bundle.

**Comment by Baldinof** [comment]
Hi!

Thank you for the report and sorry for the late response :)

symfony/psr-http-message-bridge should handle streamed response : https://github.com/symfony/psr-http-message-bridge/blob/master/Factory/PsrHttpFactory.php#L133

After investigations, it turns out that Symfony has a [StreamedResponseListener](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/EventListener/StreamedResponseListener.php#L41) that sends the response before the `Kernel::handle()` is done...

Then when symfony/psr-http-message-bridge tries to convert it into a PSR response, [the callback is not called again](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/StreamedResponse.php#L99).

I wrote another PR to fix this, if you want to review it : https://github.com/Baldinof/roadrunner-bundle/pull/11

#10 should have fixed this too but I prefer to limit code in `KernelHandler`.

**Comment by vsychov** [comment]
Thanks, @Baldinof , you PR looks much better, can you merge it please?

**Comment by Baldinof** [comment]
Done, and released with 1.2.2 :)
